dmotion:
  ankle_x:
    data: [0, 1.2000000000000002, 1.9927848527079626, 2.0045187843222791, 2.2369547741925153, 3.5000000000000111, 4.1627185518345184, 5.5, 0, 0, -15.990460391935018, 9.4222064349076557, 21.055085860216277, 0.8199999999999984, -0.03573839730182371, 0, 0, 0, 1.4495825228425676, 95.489475171068136, -1.1917535925942069, -9.5143644542220187, 0, 0]